packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-checkbox-group/example.html

```diff
@@ -0,0 +1,238 @@
+<form onchange="notify(event)" oninput="notify(event)">
+  <gux-form-field-checkbox-group-beta>
+    <label slot="group-label">Food</label>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="pizza" />
+      <label slot="label">Pizza</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="pasta" checked />
+      <label slot="label">Pasta</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="hamburger" />
+      <label slot="label">Hamburger</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input
+        slot="input"
+        type="checkbox"
+        name="food-1"
+        value="sandwich"
+        disabled
+      />
+      <label slot="label">Sandwich</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="sushi" />
+      <label slot="label">Sushi</label>
+      <span slot="error">Subject to availibility</span>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="spaghetti" />
+      <label slot="label">Spaghetti</label>
+      <span slot="help">This is a help message</span>
+    </gux-form-field-checkbox>
+  </gux-form-field-checkbox-group-beta>
+</form>
+
+<h2>Example With Nested Checkboxes</h2>
+<form onchange="notify(event)" oninput="notify(event)">
+  <gux-form-field-checkbox-group-beta>
+    <label slot="group-label">Food</label>
+    <gux-form-field-checkbox slot="group-checkbox">
+      <input slot="input" type="checkbox" />
+      <label slot="label">No restrictions</label>
+    </gux-form-field-checkbox>
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="pizza" />
+      <label slot="label">Pizza</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="pasta" checked />
+      <label slot="label">Pasta</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="hamburger" />
+      <label slot="label">Hamburger</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="sushi" />
+      <label slot="label">Sushi</label>
+      <span slot="error">Subject to availibility</span>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="spaghetti" />
+      <label slot="label">Spaghetti</label>
+      <span slot="help">This is a help message</span>
+    </gux-form-field-checkbox>
+  </gux-form-field-checkbox-group-beta>
+</form>
+
+<h2>Example With Nested Checkboxes and disabled checkbox</h2>
```

Comment:
This is kind of nitpicky but I think this title could be 'Example 3 with Nested Checkboxes & Mixed State'. 

Also, the numbering of the examples seems a bit out now.